.github/workflows/ci-maven.yaml

```diff
@@ -22,9 +22,12 @@ jobs:
       - run: java -version
       - name: Unit Tests
         run: |
-          mvn install --batch-mode --no-transfer-progress -Dcheckstyle.skip \
+          mvn test --batch-mode --no-transfer-progress -Dcheckstyle.skip \
```

Comment:
`-DenableTestCoverage `?

Reply:
I see! thanks for clarifying. That check will be included as part of the sonar job in the follow-up PR: https://github.com/googleapis/gapic-generator-java/pull/1482/